main.go

```diff
@@ -34,7 +34,6 @@ type metricConfiguration struct {
 type configuration struct {
 	BaseURL     string                `yaml:"baseURL"`
 	Login       string                `yaml:"login"`
-	Password    string                `yaml:"password"`
```

Comment:
Please keep the password still in there but set it to `yaml:"-"` :)

Reply:
Ah well, makes sense :)